src/Wrapper.js

```diff
@@ -166,10 +172,14 @@ class Wrapper extends React.Component {
       walletWeb3,
       wrapper,
       appsLoading,
-      locator: { instanceId, params },
+      locator: { instanceId, params, dao },
+      isDaoDeprecated,
     } = this.props
+
     return (
-      <React.Fragment>
+      <ModalProvider>
+        <ModalView />
+        {isDaoDeprecated && <DeprecatedBanner dao={dao} />}
```

Comment:
I think the abstraction for the "notice banner" might not be at the right level. Given that we might reuse this "notice banner" later on, and have messages other than this deprecation notice, it might make sense to use a generic banner container here that we then render a body onto, rather than encapsulating it all into `DeprecatedBanner` as we are now.

Maybe components like `NoticeBannerContainer` and `GenericNoticeBannerBody` (that `DeprecatedBannerBody` would use) would fit?

Reply:
Originally I added a `BannerView` (similar to the `ModalView` placeholder), yet there is no specific use-case for it so with @bpierre we thought it best to hardcode `DeprecatedBanner` for the time being, until we have the requirement to add another banner.

Reply:
Sounds good! Let's refactor this later if/when the need arises.